perf_test/blas/blas3/KokkosBlas3_gemm_perf_test.hpp

```diff
@@ -1116,6 +1116,106 @@ void __do_gemm_parallel_experiment6(options_t options, gemm_args_t gemm_args) {
   return;
 }
 
+template <class MemberType, class SimdViewType, class TransAType, class TransBType,
+          class BlockingType>
+class parallel_batched_gemm_experiment7 {
+ private:
+  SimdViewType &A, &B, &C;
+  gemm_args_t gemm_args;
+
+ public:
+  parallel_batched_gemm_experiment7(SimdViewType &_A, SimdViewType &_B,
+                                    SimdViewType &_C, gemm_args_t _gemm_args)
+      : A(_A), B(_B), C(_C), gemm_args(_gemm_args) {}
+
+  KOKKOS_INLINE_FUNCTION
+  void operator()(const MemberType &member) const {
+    auto i = member.league_rank();
+    Kokkos::parallel_for(Kokkos::ThreadVectorRange(member, A.extent(0)),[&](const int &vector_lane) {
+	auto svA = Kokkos::subview(A, vector_lane, Kokkos::ALL(), Kokkos::ALL(), i);
+	auto svB = Kokkos::subview(B, vector_lane, Kokkos::ALL(), Kokkos::ALL(), i);
+	auto svC = Kokkos::subview(C, vector_lane, Kokkos::ALL(), Kokkos::ALL(), i);
+
+	KokkosBatched::TeamGemm<MemberType, TransAType, TransBType, BlockingType>::invoke(member, gemm_args.alpha, svA, svB, gemm_args.beta, svC);
```

Comment:
@e10harvey Can we use the selective interface ```Gemm``` instead? It is in here https://github.com/kokkos/kokkos-kernels/blob/77848fd5552db0a89287f29f6c5d9faba22a02d1/src/batched/KokkosBatched_Gemm_Decl.hpp#L83-L106
It allows us to choose either ```SerialGemm``` (running on host) or ```TeamGemm``` (running on GPU) with ```Mode```. This is useful when we want to use ```Algo::Gemm::CompactMKL``` with ```SerialGemm``` interface

Reply:
Can you also add an option for CompactMKL too?